### PR TITLE
Assignment 1 question 1 solution

### PR DESCRIPTION
**Pull request by MK-09-coder**

### CODE REVIEW

File: Royaleagle73/A1_Q1.c [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+}
+int main()
+{ 
+int a[5]={1,2,3,4,5},n=2,i;
+replace_cube(a,n);
+for(i=0;i<5;i++)
+{
+printf("%d\t",a[i]);
+}    
+return 0;
```
Comment (RoyalEagle73) [inline Royaleagle73/A1_Q1.c]: Can you please fix formatting of code for better readability. Also, add a new line at the end of file.

**Comment by RoyalEagle73** [comment]
Also, add screenshot showing the output in description.

File: Royaleagle73/A1_Q1.c (hunk 1 above)
Comment (MK-09-coder) [reply Royaleagle73/A1_Q1.c]: output

Comment (MK-09-coder) [reply Royaleagle73/A1_Q1.c]: Kindly check the new commit i made on the PR,thank you.

Comment (MK-09-coder) [reply Royaleagle73/A1_Q1.c]: under "Files changed" my updated code is avaliable

**Pull request closed without merging**